### DISCUSSION TITLE
Upload the app on f-droid (or the IzzyOnDroid f-droid repository if that's simpler)

### DISCUSSION THREAD

**Discussion by ikcii** [axkr/symja_android_library#1057] [Ideas]
F-Droid is a great platform for open source apps of all kinds, I believe the app could gain some more traction in the open source community if it can be downloaded there. Being able to download it easily without needing proprietary google services is also an option a lot of people enjoy having

**Comment by axkr** [comment]
If someone is interested, the following app can be published on F-Droid.
- https://github.com/axkr/symja-example

**Reply by ikcii** [reply]
By the way, I understand "Calculator N+" app on Play Store that is linked in the readme of this repository uses this library?

(this one: https://play.google.com/store/apps/details?id=com.duy.calculator.free)

If yes, I believe that means the app is also licensed under GPL-3 since from what I understand that's the way that license works?

Also, do you know where I can find any other information about that app and it's developer? There are no links and no information on the developer of that app that I can find.
